### ISSUE TITLE
(Feature Request)  Ability to Concatenate

### ISSUE THREAD

**Issue by rjm831** [devhyper/open-video-editor#16]
Title says it all.  Would love to see the app have the ability to merge two or more files together.  Thanks for developing a great app!

**Comment by PaweX** [comment]
Yes, I downloaded the app just for this feature. The project looks promising, hopefully the feature will appear soon ;).

**Comment by devhyper** [comment]
Will come once composition previewing is supported in Media3: https://github.com/androidx/media/issues/1014

**Comment by rjm831** [comment]
Appreciate the development.  Thank you!

**Comment by yareckon** [comment]
It looks like the upstream library just marked the (followup) issue as completed:  https://github.com/androidx/media/issues/1488

**Comment by yareckon** [comment]
Trimming videos and then concatenating them together for export is in my opinion, the MVP for a video editing app :)

**Comment by maxstarkenburg** [comment]
Chiming in to mention I would love this feature too in a FOSS video editor. I'm currently stuck having to manually rename files, list them in a text file, then running ffmpeg in Termux to achieve video concatenation. Would love a (privacy-respecting) Android app that could simplify all these steps and save me some time.
